ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity.component.ts

```diff
@@ -382,7 +382,7 @@ export class ActivityComponent extends BaseActivityComponent implements OnInit,
 
     private sendActivityAnalytics(): void {
         this.sendConsentAnalytics();
-        if (this.model.name) {
+        if (this.model.title) {
             this.analytics.emitCustomEvent(GoogleAnalytics.SubmitSurvey, this.model.activityCode);
         }
```

Comment:
The Prequal in Osteo was defined to have an empty title (I believe this was done to fit the UI design). Since empty strings are falsy in JavaScript, will we miss out on analytic events for Osteo's Prequal?

Reply:
Good catch!
Yes, we will miss analytic events, I think we need to create a ticket to fix it because I saw one more issue here with GA.

Reply:
DDP-4396